src/language_features/call_hierarchy.rs

```diff
@@ -32,56 +44,77 @@ fn request_call_hierarchy(
     meta: EditorMeta,
     ctx: &mut Context,
     incoming_or_outgoing: bool,
-    result: Option<Vec<CallHierarchyItem>>,
+    results: Vec<(ServerName, Option<Vec<CallHierarchyItem>>)>,
 ) {
+    let result = results
+        .into_iter()
+        .find(|(_, response)| response.is_some())
+        .and_then(|(server_name, item)| Some((server_name, item.unwrap())));
```

Comment:
I don't think we should call `unwrap()` here, at least the previous code didn't.
Of course in practice, most language servers will probably send an empty vector but I don't think the protocol guarantees that.
I'd write something like

	and_then(|...| item.into_iter().map(|item| (server_name, item)));